model/project_ref.go

```diff
@@ -1181,6 +1181,11 @@ func GetTasksWithOptions(projectName string, taskName string, opts GetProjectTas
 		task.DisplayNameKey: taskName,
 		task.StatusKey:      bson.M{"$in": finishedStatuses},
 	}
+	if opts.MainlineOnly {
+		match[task.RequesterKey] = bson.M{"$in": evergreen.SystemVersionRequesterTypes}
+	} else if opts.PatchOnly {
+		match[task.RequesterKey] = evergreen.PatchVersionRequester
```

Comment:
I think this should also cover the `IsGitHubPatchRequester` case

Reply:
updated to take in a list of requesters